### PR TITLE
Add set-flag command

### PR DESCRIPTION
**Pull request by kirillgarbar**
New command is added to the table command group.
It sets flag inside table's data directory.

### CODE REVIEW

File: ch_tools/chadmin/cli/table_group.py [hunk 1]
```diff
@@ -363,3 +369,59 @@ def materialize_ttl_command(ctx, dry_run, all_, database, table, exclude_table):
...
+    Create a flag with the specified name inside the data directory of the table.
+    """
+    if database is None:
+        return
+
+    tables = (
+        get_tables_to_convert(ctx, database, table, exclude_table)
+        if flag == CONVERT_TO_REPLICATED_FLAG
+        else get_tables_dict(ctx, database, table, exclude_table)
+    )
```
Comment (kirillgarbar) [inline ch_tools/chadmin/cli/table_group.py:417]: We can also make this command more generic and make user's responsibility to filter tables with not MergeTree engine, but currently there are no other table flags in clickhouse other than convert_to_replicated.

**Comment by kirillgarbar** [comment]
I am also not sure if we should enable this feature until convertation fix is released, currently it may not work with tables that do not use default storage policy

File: ch_tools/chadmin/cli/table_group.py [hunk 2]
```diff
@@ -363,3 +370,67 @@ def materialize_ttl_command(ctx, dry_run, all_, database, table, exclude_table):
...
+    exclude_table: str,
+    flag: str,
+    verbose: bool,
+) -> None:
+    """
+    Create a flag with the specified name inside the data directory of the table.
+    """
+    if database is None:
+        print("Database is not specified")
+        return
```
Comment (Alex-Burmak) [inline ch_tools/chadmin/cli/table_group.py:413]: Can be replaced with `required=True` in `@option` definition.

But I suggest to do make the check less strict: 
```
    if not any((all_, database, table)):
        ctx.fail(
            "At least one of --all, --database, --table options must be specified."
        )
```

File: ch_tools/chadmin/cli/table_group.py [hunk 3]
```diff
@@ -363,3 +370,67 @@ def materialize_ttl_command(ctx, dry_run, all_, database, table, exclude_table):
...
+@option(
+    "--table",
+    type=StringParamType(),
+    help="Filter in tables by the specified table name.",
+)
+@option(
+    "--exclude-table",
+    type=StringParamType(),
+    help="Filter out tables by the specified table name.",
+)
```
Comment (Alex-Burmak) [inline ch_tools/chadmin/cli/table_group.py:390]: Suggest to also add options `--exclude-database` and `--all`.

File: ch_tools/chadmin/internal/table.py [hunk 4]
```diff
@@ -183,3 +189,36 @@ def materialize_ttl(ctx, database, table, echo=False, dry_run=False):
...
+    )
+    return [
+        {
+            "database": item["database"],
+            "table": item["table"],
+            "data_paths": item["data_paths"],
+            "engine": item["engine"],
+        }
+        for item in tables
+    ]
```
Comment (Alex-Burmak) [inline ch_tools/chadmin/internal/table.py:224]: Looks like a redundant logic. Why do not return `list_tables` result as is?

File: ch_tools/chadmin/cli/table_group.py (hunk 1 above)
Comment (Alex-Burmak) [reply ch_tools/chadmin/cli/table_group.py:426]: I like this idea. Let's do it.

And in this case I'd suggest to replace `--flag` `@option` with mandatory `@argument`.

Then setting the flag `convert_to_replicated` for all tables will look something like this: 
```
chadmin table set-flag --all --exclude-database system,_system --engine %MergeTree --exclude-engine Replicated% convert_to_replicated
```

**Review by Alex-Burmak — CHANGES_REQUESTED** [review]

**Comment by Alex-Burmak** [comment]
> I am also not sure if we should enable this feature until convertation fix is released, currently it may not work with tables that do not use default storage policy

I think it's OK to not wait for the fix.

**Review by Alex-Burmak — APPROVED** [review]

**Pull request merged**